.github/workflows/linters.yml

```diff
@@ -32,7 +32,3 @@ jobs:
         python -m pip install --upgrade pip
         python -m pip install nose nose-exclude coverage coveralls
         pip install --progress-bar off .
-    - name: Lint with Ruff
-      uses: astral-sh/ruff-action@v3
-      run: |
-        ruff check --output-format=github
```

Comment:
File `.github/workflows/linters.yml` is now useless as it is. Do I remove it entirely, or keep it to add coverage tests later on?